eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -1,157 +1,9 @@
 parameters:
-- name: AdditionalPlatforms
-  type: object
-  default: {}
-- name: Clouds
-  type: string
-  default: 'Public'
-- name: SupportedClouds
-  type: string
-  default: 'Public'
-- name: UnsupportedClouds
-  type: string
-  default: ''
-- name: PreSteps
-  type: object
+- name: Stages
+  type: stageList
   default: []
-- name: PostSteps
-  type: object
-  default: []
-- name: EnvVars
-  type: object
-  default: {}
-- name: MaxParallel
-  type: number
-  default: 0
-- name: BuildInParallel
-  type: boolean
-  default: true
-- name: TimeoutInMinutes
-  type: number
-  default: 60
-- name: Location
-  type: string
-  default: ''
-- name: SDKType
-  type: string
-  default: all
-- name: ServiceDirectory
-  type: string
-  default: not-specified
-- name: TestResourceDirectories
-  type: object
-  default:
-- name: Project
-  type: string
-  default: '**'
-- name: TestSetupSteps
-  type: stepList
-  default: []
-- name: DeployTestResources
-  type: boolean
-  default: true
-- name: CloudConfig
-  type: object
-  default:
-    Public:
-      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-    Preview:
-      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
-    Canary:
-      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-      Location: 'eastus2euap'
-    UsGov:
-      SubscriptionConfiguration: $(sub-config-gov-test-resources)
-    China:
-      SubscriptionConfiguration: $(sub-config-cn-test-resources)
-- name: MatrixConfigs
-  type: object
-  default:
-    - Name: NET_live_test_base
-      Path: eng/pipelines/templates/stages/platform-matrix.json
-      Selection: sparse
-      GenerateVMJobs: true
-- name: AdditionalMatrixConfigs
-  type: object
-  default: []
-- name: MatrixFilters
-  type: object
-  default: []
-- name: MatrixReplace
-  type: object
-  default: []
-- name: PlatformPreSteps
-  type: object
-  default: []
-- name: PlatformPostSteps
-  type: object
-  default:
-    - task: CopyFiles@2
-      condition: and(succeededOrFailed(), eq(variables.TestMode, 'Record'))
-      displayName: "Copy Test Recordings"
-      inputs:
-        sourceFolder: '$(Build.SourcesDirectory)'
-        # NOTE: ServiceDirectory cannot be passed as a compile time value, because it cannot be rendered
-        # in the context of a parameter definition.
-        contents: 'sdk/$(ServiceDirectory)/**/SessionRecords/**/*.json'
-        targetFolder: '$(Build.ArtifactStagingDirectory)/SessionRecords'
-    - task: PublishBuildArtifacts@1
-      condition: and(succeededOrFailed(), eq(variables.TestMode, 'Record'))
-      displayName: "Publish Test Recordings"
-      inputs:
-        pathToPublish: '$(Build.ArtifactStagingDirectory)/SessionRecords'
-        artifactName: SessionRecords
 
 variables:
```

Comment:
Lets see if we can setup a template pararmeter that can exclude these so we can use this via extends or import.